examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -189,10 +204,22 @@ export interface IBaseDocument extends IEventProvider<IBaseDocumentEvents> {
 	 * Add a task list with a specific id.
 	 */
 	readonly addTaskList: (props: IBaseDocumentInitialState) => void;
+
 	/**
 	 * Get the task list with the specified ID.
 	 */
 	readonly getTaskList: (id: string) => ITaskList | undefined;
+
+	/**
+	 * Gets the client ID of the current leader
+	 */
+	readonly getLeader: () => string | undefined;
```

Comment:
What does it mean for this to return `undefined`? Indicates that no client is leader, right? We should document this.